src/components/views/rooms/NotificationBadge/StatelessNotificationBadge.tsx

```diff
@@ -28,7 +28,10 @@ interface Props {
     count: number;
     level: NotificationLevel;
     knocked?: boolean;
-    type?: "badge" | "dot";
+    /**
+     * If true, the badge will always be displayed as a dot. Count will be ignored.
```

Comment:
`count` is not entirely ignored: `mx_NotificationBadge_visible` is set if it is zero (and symbol is null), but not if it is non-zero. I don't understand *why* that is the logic, but it seems to be the case?

Also: "the badge will always be displayed as a dot" -> "where we would normally show a badge, we instead show a dot", I think?

It would also be helpful to explain here, somewhere, WTF we mean by a "badge" and how it differs from a "dot".

Reply:
Yeah, I don't fully understand the logic of how count affects things. In 3d649865a9 I've just changed it to be more general so at least it isn't lying. Hopefully the other doc clears things up.